internal/resource/protoc-gen-resource-types/internal/generate/generate.go

```diff
@@ -132,27 +127,36 @@ var (
 
 package {{.PackageName}}
 
+{{if ne .PackageName "pbresource" -}}
```

Comment:
There are two updates to this template:

1. the generated code now keeps all consts, vars and functions for the kind together instead of grouping all consts, then vars then functions.
2. Checking if resource type being generated lives within the pbresource package. This was needed to handle generating the code for the `Tombstone` resource in that package.